docs/builtins/Keysets/enforce-keyset.md

```diff
@@ -0,0 +1,40 @@
+## enforce-keyset
+Use `enforce-keyset` to execute a specified `GUARD` or a defined keyset named `KEYSETNAME` to enforce the desired predicate logic.
+
+### Basic syntax
+
+To execute a `GUARD` or a defined keyset to enforce desired predicate logic, use the following syntax:
```

Comment:
As a newcomer to the language, I might not understand what "execute a keyset" means. How do you reckon
```suggestion
To enforce a `GUARD`'s predicate or a defined keyset, use the following syntax:
```
(it's also shorter!)